.gitattributes

```diff
@@ -0,0 +1,2 @@
+/src/**/* linguist-generated=true
+/bindings/**/* linguist-generated=true
```

Comment:
The tree-sitter-* repos that live in the tree-sitter org do `linguist-vendored` instead. Is there any advantage to this over that? I've never even seen a `.gitattributes` file before :laughing: 

https://github.com/tree-sitter/tree-sitter-javascript/blob/fdeb68ac8d2bd5a78b943528bb68ceda3aade2eb/.gitattributes

Reply:
I'm not sure what the difference is, I think generated changes some of the text for the collapsed diffs to "generated files are not shown" or something, but this might happen with vendored as well. I think it makes sense to stay in line with what `@tree-sitter` repos are doing :+1: